packages/application-shell/src/from-core/deprecate-component/deprecate-component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
I wouldn't mind not having `from-core` in the path. We can duplicate this without feeling "bad" I guess 😄.

Reply:
😇

Where should I put it then? It's not really an AppShell specific component, it can also be used by other packages...

Reply:
Oh sorry. I wasn't clear. Keep it in this repo. I just found the `from-core` path strange. I'd put it where it's needed to into the `applications-shell` package. I just got confused by having a new package called `from-core`.

Reply:
Ah ok. Well, the thing with those components is that they are copy-pasted from core. So in case we need to change something there, we should also replicate the changes in core (and the other way around). I think until we find a more maintainable solution (e.g. some components into ui-kit, other components somewhere else) we should keep it like this so we know which components need to be migrated somewhere.

Reply:
I like that we are documenting that they are duplicates from `core`. However I would add a comment to the file which explains this instead of using the folder hierarchy to communicate it. This allows us to place the components in the folder which fits them best and makes it even more obvious that they're duplicates.